scripts/basic-workflow-tests.sh

```diff
@@ -114,7 +114,7 @@ test_init() {
 
     step Checking config.yaml
     if [ "${MIXTRAL}" -eq 1 ]; then
-        sed -i -e 's/models\/merlinite.*/models\/mixtral-8x7b-instruct-v0\.1\.Q4_K_M\.gguf/' "${CONFIG_HOME}/config.yaml"
+        sed -i -e 's/models\/merlinite.*/models\/mixtral-8x7b-instruct-v0\.1\.Q4_K_M\.gguf/' "${CONFIG_HOME}/instructlab/config.yaml"
```

Comment:
Is line 113 still valid though? I mean the `[ -f config.yaml ]`, does it need to be updated too with `"${CONFIG_HOME}/instructlab/config.yaml"`?

Reply:
That's a good point, it would evaluate to false since the e2e script is cannot possible be executed in the temp directory as it's something we create during execution.

Reply:
not related to this PR, but I'll go ahead and fix it.

Reply:
done.